cli/src/commands/root.rs

```diff
@@ -22,6 +22,8 @@ use crate::command_error::CommandError;
 use crate::ui::Ui;
 
 /// Show the current workspace root directory
+///
+/// Synonymous with `jj workspace root`.
```

Comment:
nit: `jj undo` says "Undo an operation (shortcut for jj op undo)", so it would be nice to be consistent with that. Putting it on a separate line as you have done works better if the first line is long. So I would suggest updating `jj undo` to match.

Reply:
Done!

Reply:
Oh, I misread, and I updated `jj root` to match `jj undo`. But I think that's fine as well? I like the `undo` wording. Or feel free to change it to whatever wording you prefer.

Reply:
Yeah, that's also fine, as long as it fit on one line (but I don't know exactly how we define that).

Reply:
Great, I force pushed, so I think this can probably be merged then.

Reply:
There are failing tests. Could you update those too? You can use `cargo insta test --review --test-runner nextest`  or just `cargo insta test --review`.

Reply:
@martinvonz Sorry about that -- I didn't even think to run the tests or check the CI failures. 😅 I've pushed again so this should be good to merge now.